.github/workflows/utility-gated-requirement-checks.yml

```
@@ -0,0 +1,48 @@
+        uses: actions/checkout@v4
+
+      - name: Check for ${{ inputs.path }} path changes
+        id: filter
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
+            ${{ inputs.path }}:
+              - '${{ inputs.path }}/**'
+
+      - name: Log path changes decision
+        run: 'echo "${{ inputs.path }} files changed: ${{ steps.filter.outputs.${{ inputs.path }} }}"'
```

actionlint (1.7.7)

got unexpected character '$' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '

(expression)